### DISCUSSION TITLE
Great component and Wonderfully documented

### DISCUSSION THREAD

**Discussion by prabhuignoto** [vue-final/vue-final-modal#33] [Q&A] [answered]
Thanks for the component. This is done very well.

I also liked the documentation site. could you tell me the tech stack used for the documentation site?
I am building a doc site myself and found your docs to be impressive.

Best
Prabhu

**Comment by hunterliu1003** [comment, marked as answer]
Hi @prabhuignoto,

The documentation using:

- [nuxt/content](https://content.nuxtjs.org/)
- [TailwindCSS](https://tailwindcss.com/)
- [@nuxtjs/google-analytics](https://google-analytics.nuxtjs.org/)
- [Deploy with Netlify](https://www.netlify.com/)

Hope these help you build a awesome website.

**Reply by prabhuignoto** [reply]
Thanks  @hunterliu1003  for the info.
